safe-defaults/tmpfiles-enable.conf.in

```diff
@@ -13,6 +13,8 @@ L @SYSCONFDIR@/opt/chrome/policies/managed/safe-search.json - - - - @DATAROOTDIR
 D /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search 0755 - - -
 D /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search/@FLATPAK_ARCH@ 0755 - - -
 L /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search/@FLATPAK_ARCH@/1 - - - - @DATAROOTDIR@/eos-safe-defaults/chrome
+D /var/local/share/ca-certificates 0755 - - -
```

Comment:
I think you should use `d` here rather than `D`. If the local administrator has placed more certificates in `/usr/local/share/ca-certificates`, we don't want them to be removed.

Reply:
ok